main_code/int_api/spreadsheet.py

```diff
@@ -103,7 +103,7 @@ def read_cell() -> str:
     log(pyautogui.hotkey, 'ctrl', 'c')
     log(pyautogui.hotkey, 'ctrl', 'f')
     log(pyautogui.hotkey, 'ctrl', 'v')
-    log(pyautogui.hotkey, 'ctrl', 'ctrl', 'a')
+    log(pyautogui.hotkey, 'shift', 'fn', 'left')
```

Comment:
Can you check to see using shift home instead of shift fn left?  Shift fn left will not have consistent behavior whereas I'm pretty sure shift home will because not all keyboards have a home button but the OS reocgnizes the input, but I think shift fn left might not be universally supported.